app/views/pg_hero/home/_live_queries_table.html.erb

```diff
@@ -33,8 +33,9 @@
           <% unless @database.filter_data %>
             <%= button_to "Explain", explain_path, params: {query: query[:query]}, form: {target: "_blank"}, class: "btn btn-info" %>
           <% end %>
-          <%= button_to "Kill", kill_path(pid: query[:pid]), class: "btn btn-danger" %>
-        </td>
+          <% if PgHero.config["show_kill"] %>
+            <%= button_to "Kill", kill_path(pid: query[:pid]), class: "btn btn-danger" %>
+          <% end %>        </td>
```

Comment:
I'd put the closing td back on its own line, just to avoid merge conflicts down the line:
```suggestion
          <% end %>
        </td>
```